startup.py

```diff
@@ -251,9 +256,20 @@ def prepare_launch(self, exec_path, args, file_to_open=None):
         required_env["SGTK_CONTEXT"] = sgtk.context.serialize(self.context)
 
         # ensure scripts are up to date on the substance painter side
-        user_scripts_path = (
-            os.path.expanduser(r"~/Documents/Allegorithmic/Substance Painter/plugins")
-        )
+
+        # Platform-specific plug-in paths
+
+        if sys.platform == 'win32':
+
+            meh= ctypes.create_unicode_buffer(ctypes.wintypes.MAX_PATH)
```

Comment:
I'd use a different variable name for this, perhaps ``path_buffer`` .

Reply:
Hey Diego,
Just modified the code with your notes.
Hope this works fine :)